woocommerce/woocommerce-android
# Issue/8381 display subscription in order details

⚠️ Depends on the FluxC counterpart https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2666
Closes: #8381 

### Description
This PRs brings read-only support for WooCommerce Subscriptions. It does that by fetching the subscription information using the order id. The code first checks that the WooCommerce Subscription extension is enabled and active before making any request. Then it will fetch the subscription information using the order id. This PR still lacks support for subscription information for Renewal Orders that follow a different pattern than regular orders.

### Testing instructions

1. Open the Orders screen
2. Open an order containing subscriptions
3. Check that the subscription information is displayed in the order details screen

### Images/gif

https://user-images.githubusercontent.com/18119390/222222859-a54c3795-ccc5-45d8-8492-1beb3d1cbc26.mp4
